lighthouse-core/closure/closure-type-checking.js

```diff
@@ -32,14 +32,20 @@ gulp.task('compile-report', () => {
   return gulp.src([
     // externs
     'closure/third_party/commonjs.js',
-    'lib/filer-namer.js',
+
+    'lib/file-namer.js',
     'report/v2/renderer/*.js',
   ])
 
   // Ignore `module.exports` and `self.ClassName = ClassName` statements.
   .pipe(replace(/^\s\smodule\.exports = \w+;$/gm, ';'))
   .pipe(replace(/^\s\sself\.(\w+) = \1;$/gm, ';'))
 
+  // Remove node-specific code from file-namer so it can be included in report.
+  .pipe(replace(/^\s\smodule\.exports = {\w+};$/gm, ';'))
```

Comment:
happens to not match the `module.exports` line in the renderer files:
```js
module.exports = ReportRenderer;
```
but yah this seems to be the right solution.